templates/jaasai/index.html

```diff
@@ -222,10 +286,27 @@ <h2>Why use Juju?</h2>
     </div>
     <div class="row">
       <div class="col-6 u-align-text--center">
-        <img src="https://assets.ubuntu.com/v1/4c62db97-why-use-juju-diagram-left.svg" alt="Diagram on 'Why use Juju (Part 1)'" />
+        <img src="" alt="Diagram on 'Why use Juju (Part 1)'" />
```

Comment:
The alt is being shown along with the image. <img width="751" alt="Screen Shot 2019-09-04 at 9 53 09 AM" src="https://user-images.githubusercontent.com/532033/64271042-dc513980-cef9-11e9-8da1-689a66e39f3f.png">